antora-ui-camel/src/css/footer.css

```diff
@@ -182,59 +181,45 @@ input[type="checkbox"] {
     width: auto;
   }
 
-  footer .footer .context a {
-    font-size: 13.5px;
-  }
-
-  footer .footer .context a::after {
-    width: 5px;
-    height: 5px;
-  }
-
   footer .footer .footer-icons .brand-icon {
     height: 1.5rem;
   }
 
-  footer .footer .show-menu {
-    float: right;
-    height: 40px;
-    margin-top: -2.5rem;
-  }
-
-  footer .footer .hide-menu {
-    float: right;
-    height: 35px;
-    margin-top: -2.5rem;
+  footer .footer input[type="checkbox"] {
+    display: inline;
+    position: relative;
+    top: 2rem;
+    left: 45vw;
+    width: 1.75rem;
+    height: 1.75rem;
+    transition: 0.2s;
   }
 
-  footer .footer .footer-menu {
-    margin-top: 1rem;
+  footer .footer input[type="checkbox"] {
```

Comment:
Is this meant to be changed as `footer .footer input[type="checkbox"]:checked` as it's the same as the stated class as above and thus the preview is failing for the same?

Reply:
Yeah I noticed a quirk artefact in Chrome so I fixed it last minute without running the builds/checks, should be okay now.